pages/neighbourhoods.py

```diff
@@ -0,0 +1,434 @@
+                      autosize= False,
+                      font = {"size": style["fontsize"], "color": style["color"]},
+                      paper_bgcolor='white', 
+                      yaxis={'categoryorder':'total ascending'}
+                      )
+
+    return title, fig
+
+@callback(
+    Output('wijk_trend_label', 'children'),
+    Output('wijk_trend_fig', 'figure'),
+    Input('map', 'clickData'),
+    Input('drop_var_id', 'value'),
+    Input('drop_municipality', 'value'),
+    Input('drop_municipality_spec_id', 'value'),
+    prevent_initial_call=False
+    )
+
+def update_graph(clickData, 
+                 xaxis_column_name, wijk_name, wijk_spec):
+    '''
+    Update line graph 
+    '''
+
+    if len(wijk_spec) == 0:
+        fig = px.line(x=[0, 10], y=[0, 0])
+        fig.update_xaxes(showticklabels=False)
+        fig.update_yaxes(showticklabels=False)
+        
+        return "No neighbourhood selected", fig    
+    
+    dff = df.query("Wijknaam in @wijk_spec")
+    
+    fig = px.line(dff, x='YEAR', y=  xaxis_column_name, color='Wijknaam', color_discrete_sequence=colorscale)
+
+    fig.update_layout(xaxis={
+        "rangeslider":{"visible": True},
+        "type": "date"
+        },font = {"size": style["fontsize"], "color": style["color"]} 
+    )
+    
+    fig.update_layout(dict(updatemenus=[
+                        dict(
+                            type = "buttons",
+                            direction = "left",
+                            buttons=list([
+                                
+                                dict(
+                                    args=["visible", True],
+                                    label="Select All",
+                                    method="restyle"
+                                ),
+                                dict(
+                                    args=[{'visible':'legendonly'} ],
+                                    label="Remove All",
+                                    method="restyle"
+                                )
+                            ]),
+                            pad={"r": 0, "t": -20},
+                            showactive=False,
+                            x=1,
+                            xanchor="right",
+                            y=1.1,
+                            yanchor="top"
+                        )
+                    ]
+              ))
+    
+    if clickData is None: #change chart based on selection from the select
+        title = '{} - {}'.format(xaxis_column_name, dff["Wijknaam"].unique()[0])
+        
+        fig.update_traces(visible="legendonly")
+
+        fig.data[0].visible=True 
+
+    else: #User can click on neighbourhoods in the map to affect the linechart. 
+        title = '{} - {}'.format(xaxis_column_name, clickData['points'][0]['hovertext'])
+        fig.update_traces(visible="legendonly") 
+        fig.data[clickData['points'][0]['pointIndex']].visible=True 
```

Comment:
I think you are right that we should change the title into much more generic. not specific neighbourhood. Because there is an option to select all neighbourhoods anyway.